src/main/java/io/SchoolManagementSystem/Department.java

```diff
@@ -0,0 +1,23 @@
+package io.SchoolManagementSystem;
+
+import java.util.ArrayList;
+
+public class Department {
+    public ArrayList<Student> involvedStudents = null;
+    public ArrayList<Teacher> involvedTeacher = null;
+    public ArrayList<Staff> involvedStaff = null;
```

Comment:
Right now, having a staff member is optional, however I think that there should always be one staff member like an Admin or something, who will be the Head of Department, we would want to specify something instead of null only for the Staff hod = new Staff()